internal/evaluator/opa.go

```diff
@@ -133,7 +133,12 @@ func (o *OpaEvaluator) Eval(ctx context.Context, resourceInfo fetching.ResourceI
 		return EventData{}, fmt.Errorf("error decoding findings: %v", err)
 	}
 
-	o.log.Debugf("Created %d findings for input: %v", len(ruleResults.Findings), fetcherResult)
+	if len(ruleResults.Findings) == 0 {
```

Comment:
What's the benefit of adding an `if` here? Isn't the `Created %d findings for input` always going to be logged?

Reply:
this is a semantic change to avoid logging `Created 0 findings for input` when we don't create any findings. no actual benefit other than that tbh.

Reply:
just to clarify (unless i'm missing smth), there `Created %d findings...` is inside an `else` block, so it won't always be logged

Reply:
I guess I don't understand the benefit of having a different log line for zero findings? You can always search for `Created 0 findings for input` and that would be the equivalent of `Evaluated resource has no findings`. Without the change, it's more obvious that `Created 7 findings for input` and `Created 0 findings for input` come from the same place.